src/extension/android/packageNameResolver.ts

```diff
@@ -31,14 +32,14 @@ export class PackageNameResolver {
      * Given a manifest file path, it parses the file and returns the package name.
      * If the package name cannot be parsed, the default packge name is returned.
      */
-    private readPackageName(manifestPath: string): Q.Promise<string> {
+    private readPackageName(manifestPath: string): Promise<string> {
         if (manifestPath) {
-            let fs = new FileSystem();
+            let fs = new FileSystemNode();
             return fs.exists(manifestPath).then(exists => {
                 if (exists) {
                     return fs.readFile(manifestPath)
                         .then(manifestContent => {
-                            let packageName = this.parsePackageName(manifestContent);
+                            let packageName = this.parsePackageName(manifestContent.toString());
```

Comment:
Why do we need to manually convert `manifestContent` to string?

Reply:
Promise based fs returns `Buffer | string` type, so this causes types mismatch.